ecs-cli/Gopkg.toml

```diff
@@ -82,4 +82,9 @@
 
 [[constraint]]
   name = "github.com/docker/cli"
-  revision = "1872bd802c697f39eb783a9b7d25e60d6fe601d6"
+  revision = "750b038707852caa10806d6f760dffbe09f9c5c1"
```

Comment:
Is there a reason why we can't track something less binding than a revision? (e.g. version)